pkg/quota/image/imagestreamimport_evaluator.go

```diff
@@ -127,29 +132,35 @@ func (c *imageStreamImportUsageComputer) Usage(object runtime.Object) kapi.Resou
 
 func (c *imageStreamImportUsageComputer) getImagesUsageIncrement(isi *imageapi.ImageStreamImport) kapi.ResourceList {
 	specRefsIncrement := resource.NewQuantity(0, resource.DecimalSI)
-	allRefsIncrement := resource.NewQuantity(0, resource.DecimalSI)
+	statusRefsIncrement := resource.NewQuantity(0, resource.DecimalSI)
 
 	for _, imageSpec := range isi.Spec.Images {
```

Comment:
In short, the increment of imagestreamimages is computed as `number_of_new_istags_added_to_spec - number_of_id_refs_from_isi_that_already_exist_in_image_streams`. The latter number will be almost always zero as you already mentioned but its cheep to compute since we already have the data.